cli.py

```diff
@@ -0,0 +1,19 @@
+import ConfigParser
+import logging
+import sys
+
+def parse_args():
+    """Debug mode with param -debug"""
+    if "-debug" in sys.argv :
+        l.setLevel(logging.DEBUG)
+
+def parse_config():
+    """Read config file and loads parameters as variables"""
+    configParser = ConfigParser.RawConfigParser()
+    configFilePath = r'./config.cfg'
+    configParser.read(configFilePath)
+    cfg = {}
```

Comment:
This `cfg` var should not be just a `dict`, but an object, using or encapsulating a `dict`, which constructor ensures the configuration parameters are present and valid.
This has not being done here to skip this decission.

Reply:
I created issue #7 for this